lib/src/ema/ema_instructions.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/widgets.dart';
+
+class EMAInstructions extends StatelessWidget {
+  const EMAInstructions({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const Column(
+      children: [
+        Text(
+          "Vamos a continuar con las tareas que hemos estado practicando.",
```

Comment:
Let's use the word "actividades" since it is more friendly than the word "tareas" and it is also used in the next section below.

```suggestion
          "Vamos a continuar con las actividades que hemos estado practicando.",
```